### PR TITLE
[Framework] A series of UI accessibility improvements

### PR DESCRIPTION
**Pull request by tidoust**
This update contains a short series to address main reported accessibility issues of the User Interface. Changes are:

- Links in text are underlined (#226)
- Menu is now keyboar accessible and "Tab/Up/Down/Home/End" key friendly (#227)
- Feature labels in tables are top-aligned when there the column spans multiple rows (#229)
- Menu button is correctly labelled (#230)
- Contrast of table headers improved (#231)
- Contrast of footer improved (#232)

Actually, the menu can now be opened and closed through the "m" key. I haven't tried to advertise that anywhere for now.

The update also fixes the URL of the about page in translations (#233)

@xfq I let you update the "alt" texts for the menu button in `template-page.zh.html`!

### CODE REVIEW

File: assets/css/theme.css [hunk 1]
```diff
@@ -58,6 +58,7 @@ p.highlight {
 a {
   color: #0062af;
   text-decoration: none;
+  border-bottom: 1px solid #767676;
```
Comment (xfq) [inline assets/css/theme.css]: What's the rationale of using `border-bottom` for text decoration?

**Comment by xfq** [comment]
Asked a question about `border-bottom`, otherwise LGTM.

(nitpick: s/keyboar/keyboard/ in the commit message.)

File: assets/css/theme.css (hunk 1 above)
Comment (tidoust) [reply assets/css/theme.css]: Well, that is a good question ;)

I confess I did not think about it when I did it. I just reused styles I found elsewhere, and that is how links are styled on W3C's Web site and in W3C specs. Now that you made me think about it, I see two reasons:

1. While `text-decoration` lets one specify a color for the underline, there is no way to set the width of the line. In particular, the goal is to highlight the link on hover through a thicker border, which cannot be done with `text-decoration` (we could use a double line, but the result is so-so)
2. The underline gets drawn closer to the text when `text-decoration` is used, while `border-bottom` leaves more space. For instance, an underline goes through letters such as "g" or "p", while the border does not. I find it slightly more readable when the line gets drawn below the letters.

Both are a matter of taste...

Comment (xfq) [reply assets/css/theme.css]: OK.

Comment (xfq) [reply assets/css/theme.css]: * `#1` can be solved by [text-decoration-width](https://drafts.csswg.org/css-text-decor-4/#text-decoration-width-property) (not implemented anywhere for the moment ;)
* I didn't know any good solution for `#2`. `text-underline-position: under;` isn't perfect either...

**Pull request merged**